doc/api/tty.md

```diff
@@ -88,6 +88,17 @@ process.stdout.on('resize', () => {
 });
 ```
 
+*Note:* On Windows resize events will be emitted only if stdin is unpaused 
+(by a call to `resume()` or by adding a data listener) and in raw mode. It can 
+also be triggered if a terminal control sequence that moves the cursor is written 
+to the screen.
+
+*Note:* On Windows when changing console height the resize event will be 
```

Comment:
1. `*Note:*` -> `*Note*:`
2. IMHO the two paragraphs could be merged:
`Also, the resize event will be signaled only if the console screen buffer height was also changed...`